apps/balancer-tools/src/components/Tabs.tsx

```diff
@@ -96,7 +104,9 @@ function TabItemTrigger({
       }}
     >
       <div className="flex items-center justify-center gap-x-3">
-        {color && <div className={cn(`w-3 h-3 rounded-full bg-${color}`)} />}
+        {itemClassNames && (
+          <div className={cn("w-3 h-3 rounded-full", itemClassNames)} />
+        )}
         {children}
```

Comment:
🤔 honestly I think we should probably only have children here and leave this badge out for whoemever needs it to add it